CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Minimum Rust version has been bumped to 1.37.0
 
+### Added
+
+- Modal keyboard motion mode for copying text
```

Comment:
I think it would be clearer if we called this a "keyboard pointer" instead of "keyboard motion". This PR is giving the cursor the semantics of the mouse pointer for some actions (or at least, that's how you should think about it).

Reply:
It's not really using the same cursor or modifying its position though, but instead it uses a completely separate one to navigate around.

I think the motion part of being able to freely move around with your cursor is the important part here, I'm not sure if I would want to name this mode after the pointer. Tmux names this copy mode for example, but it can do more than just copying so I didn't want to be limited to it. We'll also likely implement search in this mode so I'm not sure if 'cursor' still sufficiently describes it at that point, or if the cursor is just a tool to help the user.

The word `motion` comes from vi here, since that's what the existing key bindings are bound to by default.